include/caffe/layer.hpp

```diff
@@ -316,6 +317,21 @@ class Layer {
     param_propagate_down_[param_id] = value;
   }
 
+  bool IsForwardPassed() const {
+    return forward_passed_;
+  }
+
+  void ForwardPassed(bool passed) {
```

Comment:
Per Google style guide and existing Caffe usage, accessors are spelled in `underscore_case`, and take the name of the member variable (so, `forward_passed_` should be accessed with a function called `forward_passed`, and set with a function called `set_forward_passed`).

This is an FYI orthogonal to the question of whether these functions should be here, which I'll now consider!